testing-common/testing-common.gradle

```diff
@@ -3,6 +3,8 @@ apply from: "$rootDir/gradle/java.gradle"
 dependencies {
   api deps.guava
   api deps.spock
+  api deps.slf4j
```

Comment:
Removed it. The reason I added these was because I have a common module that depends on testing-commons for tests that is used by 2 other modules which just extend test class from the base module. Groovy compile fails because in the class hierarchy of the test class some super class uses logback & slf4j. To me it seemed weird that just because I have an extra module I have to add additional dependencies. Perhaps adding CompileStatic to test class would also work, but I resorted to adding `api deps.testLogging` to the common module.